### PR TITLE
docs(blog): add entry about custom PIT reporter for dashboard

### PR DESCRIPTION
**Pull request by mthmulders**

### CODE REVIEW

**Comment by hugo-vrijswijk** [comment]
I think you still need to fix the formatting :). If you want, you could also add a [canonical url to your website](https://github.com/facebook/docusaurus/issues/2603#issuecomment-896135382)

File: blog/2022-08-15-connect-pit-to-stryker-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+slug: connect-pit-to-stryker-dashboard
+title: 'Connect PIT to Stryker Dashboard'
+author: Maarten Mulders
+author_title: Stryker Team
+author_url: https://maarten.mulders.it/
+author_image_url: https://maarten.mulders.it/img/twitter-profile-202106.png
+tags: [stryker, pit, java, maven]
```
Comment (hugo-vrijswijk) [inline blog/2022-08-15-connect-pit-to-stryker-dashboard.md:8]:
```suggestion
tags: [stryker, pit, java, maven, dashboard]
```

**Comment by mthmulders** [comment]
> I think you still need to fix the formatting :).

[I think the Contributing guide should mention that](https://github.com/stryker-mutator/stryker-mutator.github.io/pull/452) ;-)

File: blog/2022-08-15-connect-pit-to-stryker-dashboard.md (hunk 1 above)
Comment (mthmulders) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md:8]: > If you want, you could also add a [canonical url to your website](https://docusaurus.io/docs/markdown-features/head-metadata)

Cool, I did not even know this existed. If I understand correctly, it lets you hint search engines as to which version of (very) similar pages is "the original" and which one(s) is/are "the derivatives"?

Could be useful, but I'm too unfamiliar with Docosaurus to figure out from the docs how to do that. I tried adding this to the front matter:

```
canonical: https://maarten.mulders.it/2022/07/mutation-testing-badge-with-pit-and-stryker-dashboard/
```

But it seems not to work as I'd expect: the browser inspector still shows

```
<link rel="canonical" href="https://stryker-mutator.io/blog/connect-pit-to-stryker-dashboard/" data-rh="true">
```

Comment (hugo-vrijswijk) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md]: In the markdown article (so below the frontmatter) if you add:

```html
<head>
  <link rel="canonical" href="https://maarten.mulders.it/2022/07/mutation-testing-badge-with-pit-and-stryker-dashboard/" />
</head>
```

That should add it from what I can see in the documentation

Comment (mthmulders) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md]: Thanks... I don't think I would've found that!

Is my interpretation of "canonical URL" correct, by the way?

Comment (hugo-vrijswijk) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md]: > Is my interpretation of "canonical URL" correct, by the way?

I think so, yeah, with it a search engine might decide to place the canonical article higher in the search results than the 'copy'

File: blog/2022-08-15-connect-pit-to-stryker-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+slug: connect-pit-to-stryker-dashboard
+title: 'Connect PIT to Stryker Dashboard'
+author: Maarten Mulders
+author_title: Stryker Team
+author_url: https://maarten.mulders.it/
```
Comment (mthmulders) [inline blog/2022-08-15-connect-pit-to-stryker-dashboard.md:6]: What do you expect here? Is it always "Stryker Team", or could it also be something else, e.g. "External Contributor", "Some Rando" or "Java Idiot"?

File: blog/2022-08-15-connect-pit-to-stryker-dashboard.md (hunk 1 above)
Comment (mthmulders) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md]: Thanks for those suggestions!

File: blog/2022-08-15-connect-pit-to-stryker-dashboard.md (hunk 2 above)
Comment (hugo-vrijswijk) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md:6]: I like Java Idiot 😉. I think in most cases we've used Stryker Team/Stryker.NET team/Stryker4s team. You're still sort of part of the team 🙂 so feel free to put there what you want

Comment (mthmulders) [reply blog/2022-08-15-connect-pit-to-stryker-dashboard.md:6]: I'll stick with "Stryker Team", in that case :-)

**Comment by mthmulders** [comment]
Should I update the date in the filename & metadata before we merge this one?

**Comment by mthmulders** [comment]
> Should I update the date in the filename & metadata before we merge this one?

Done.

**Comment by hugo-vrijswijk** [comment]
🚀 Awesome!

**Pull request merged**